src/index.css

```diff
@@ -176,6 +176,126 @@
     -webkit-background-clip: text;
     -webkit-text-fill-color: transparent;
   }
+
+  .h1_32_40_heavy {
```

Comment:
I see can see a different naming here:
1) `.h1_32_40_heavy`
2) `.mini_heavy`
3) `.text_12_16_roman`

And I'd like to suggest another naming instead:

`text-h1-heavy`
`text-h2`
`text-mini-italic`

First part to indicate this is a class for text
Second part is a size name from design. In design h1, h2, mini already means size, so it should be safe to omit 16/24 exact part.
And third part is for font-style property: heavy, italic. I think better to omit `roman` because it means "normal" and the font is already normal by default. Design has "itali**a**c", you renamed it to "italic" I guess for the same reason: in web dev "italic" word is more common.

Reply:
updated